src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitSchedulerService.kt

```diff
@@ -234,6 +233,8 @@ class VisitSchedulerService(
       )
     )
 
+    visitEntity.reference = referenceService.createReference(visitEntity.id)
```

Comment:
Take a look at PrePersist on the visit entity:

  @PrePersist
  fun encodeId() {
    reference = QuotableEncoder(minLength = 8).encode(id)
  }

Reply:
awesome, works great

Reply:
awesome, works great